pyproject.toml

```
@@ -22,7 +22,7 @@ dependencies = [
     "scipy",
     "imageio",
     "torch>=1.10.1",
-    "invertedai",
+    "invertedai==0.0.16",
```

```suggestion
    "invertedai",
```
I think this will work with older versions of `invertedai` now as well. We can set this constraint in `torchdriveenv` instead.